err_stash.py

```diff
@@ -214,6 +221,10 @@ def create_plans(
         slug = repo["slug"]
         project = repo["project"]["key"]
         branches = list(stash_api.fetch_branches(project, slug, branch_text))
+        if exactly_branch_name:
```

Comment:
I think you need this check on the Github part as well.

Reply:
Due performance reasons, the search in github is always with the exactly branch name.